src/main/java/com/joyent/manta/monitor/MantaOperationContext.java

```diff
@@ -151,13 +151,14 @@ public Stopwatch getStopWatch() {
         return (Stopwatch)get(STOPWATCH_KEY);
     }
 
-    public MantaOperationContext setRequestPutHistogramsKey(final Map<String, Histogram> histogramMap) {
-        put(REQUEST_PUT_HISTOGRAMS_KEY, histogramMap);
+    public MantaOperationContext setRequestPutHistogramsMap(final Map<String,
+            Histogram> histogramMap) {
+        put(REQUEST_PUT_HISTOGRAMS_MAP, histogramMap);
         return this;
     }
 
-    public Map<String, Histogram> getRequestPutHistogramsKey() {
-        return (Map<String, Histogram>)get(REQUEST_PUT_HISTOGRAMS_KEY);
+    public Map<String, Histogram> getRequestPutHistogramsMap() {
```

Comment:
This change makes sense because you aren't returning a key. Although, you don't need to write `Map` because that is understood from the type signature.

Reply:
Okay. Will change it to ```getRequestPutHistograms``` and ```setRequestPutHistograms```